style.css

```diff
@@ -168,11 +319,206 @@ article h1 {
+  width: 100%;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.item3 {
+  box-sizing: border-box;
+  display: grid;
+  grid-template-rows: auto auto;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background-color: #fff;
+}
+
+.item3a img {
+  width: 100%;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.item4 {
+  box-sizing: border-box;
+  display: grid;
+  grid-template-rows: auto auto;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background-color: #fff;
+  flex-wrap: wrap;
+}
+
+.item4a img {
+  width: 100%;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.grid-container2 {
+  left: 375px;
+  background: #fff;
+  border-top-right-radius: 120px;
+  display: grid;
+  grid-template-columns: auto;
+}
+
+.about-me {
+  display: flex;
+  padding: 29px;
+  flex-direction: column;
+  padding-top: 100px;
+  margin: 0;
+}
+
+.about-me h1 {
+  color: #172b4d;
+  font-family: 'poppins' Arial, Helvetica, sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  margin: 0;
+}
+
+.about-me p {
+  color: #344563;
+  font-family: 'Poppins' Verdana, Geneva, Tahoma, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+}
+
+.lnks {
+  box-sizing: border-box;
+  color: #505f79;
+  padding: 0;
+}
+
+.btn {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 10px;
+  width: 134px;
+  height: 38px;
+  left: 0;
+  border: 1px solid #6070ff;
+  background-color: #fff;
+  border-radius: 8px;
+}
+
+.links i {
+  color: #505f79;
+  font-size: 50px;
+}
+
+.about-me ul {
+  display: flex;
+  list-style: none;
+  margin-left: 17px;
+}
+
+.footer {
+  background: #6070ff;
+  border-top-left-radius: 100px;
+  display: flex;
+  height: 779px;
+  margin-top: 10%;
+}
+
+.skills ul {
+  list-style: none;
+}
+
+.lang {
+  height: 24px;
+  right: 78px;
+  top: calc(50% - 24px / 2);
+  font-family: 'Poppins' Verdana, Geneva, Tahoma, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  color: #000;
+  padding: 0;
+}
+
+.frame {
+  font-family: 'Poppins' 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
+}
+
+.skl {
+  font-family: 'Poppins' Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
+}
+
+.sub-lang img {
+  width: 48px;
+  height: 48px;
+  left: 12px;
+  background: url(images.jpg);
+  border: 1px solid #fff;
+  align-items: center;
+  margin-top: 10px;
+  padding: 0;
+}
+
+.language ul li {
+  display: flex;
+  margin: 36px 0;
+  flex-direction: row;
+}
```

Comment:
- [ ] Instead of the last code reviewer helped you with the correct code to fix the issue you didn't implement it, please 🙏🏽 fix this section to be similar to the Figma template and make sure to correct all the issues before submitting for a new code review take in mind you have limited chances to approve the project. 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/212356776-b42824c6-1f5f-43ea-be65-7b0dfa443cf4.png)